src/card/card-actions.jsx

```diff
@@ -54,9 +54,10 @@ const CardActions = React.createClass({
   render() {
     let styles = this.getStyles();
 
-    let children = React.Children.map(this.props.children, (child) => {
+    var _this = this;
+    var children = React.Children.map(this.props.children, function (child) {
```

Comment:
Yes that's way better. Using the arrow function allows this to be preserved so that I don't have to write `var _this = this;` that triggers the travis-ci build error anymore.